### PR TITLE
adjust the delegated resolution handling

### PR DESCRIPTION
**Pull request by osoukup**
I first understood OSIDB-1406 wrong and this fixes it - the duplicate and migrated resolutions are now ignored

### CODE REVIEW

**Comment by osoukup** [comment]
@dmonzonis this reshuffles a bit your code created in https://github.com/RedHatProductSecurity/osidb/pull/456/files to address the comment from Przemyslaw in OSIDB-1406. Could you review the change please? Sorry for the confusion.

File: osidb/tests/test_flaw.py [hunk 1]
```diff
@@ -404,29 +422,6 @@ def test_delegated_affects(self):
...
-            type=Tracker.TrackerType.BUGZILLA,
-        )
-        TrackerFactory(
-            affects=(affect,),
-            status="closed",
-            resolution="won't do",
-            embargoed=affect.flaw.is_embargoed,
-            type=Tracker.TrackerType.BUGZILLA,
-        )
-        assert affect.delegated_resolution == Affect.AffectFix.WONTFIX
```
Comment (dmonzonis) [inline osidb/tests/test_flaw.py:428]: We could still leave this test, as it still checks that migrated/duplicate resolutions are ignored when in conjunction with another valid resolution

**Review by dmonzonis — APPROVED** [review]
LGTM, just one comment

File: osidb/tests/test_flaw.py (hunk 1 above)
Comment (osoukup) [reply osidb/tests/test_flaw.py:428]: @dmonzonis I am not removing it, I just moved the trackers to be ignored at the very top of the method to make sure that they get ignored all the time till the end. Does it make sense?

Comment (dmonzonis) [reply osidb/tests/test_flaw.py:428]: You're right, my bad, all good then!

**Pull request merged**